src/components/atoms/stories/Dialog.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { ComponentProps } from 'react';
+import { Story, Meta } from '@storybook/react';
+import Dialog from '../Dialog';
+
+export default {
+  title: 'Components/atoms/Dialog',
+  component: Dialog,
+} as Meta;
+
+const Template: Story<ComponentProps<typeof Dialog>> = (args) => <Dialog {...args} />;
+
+export const DialogS = Template.bind({});
```

Comment:
it is not possible to display the dialog from the main content area. can you include a button which will open it?